### ISSUE TITLE
Migration fails with Undefined index: file

### ISSUE THREAD

**Issue by mmccaff** [laravel/telescope#242]
Running the migration after adding the package ands installing, it gets an Undefined index: file in src/Watchers/QueryWatcher.php, line 78.

return ! Str::contains($frame['file']),

**Comment by CrixuAMG** [comment]
Same error when upgrading using `composer update`

```
Package manifest generated successfully.
ErrorException {#947
  #message: "Undefined index: file"
  #code: 0
  #file: "./vendor/laravel/telescope/src/Watchers/QueryWatcher.php"
  #line: 78
  #severity: E_NOTICE
  trace: {
    ./vendor/laravel/telescope/src/Watchers/QueryWatcher.php:78 { …}
    Laravel\Telescope\Watchers\QueryWatcher->Laravel\Telescope\Watchers\{closure}() {}
    ./vendor/laravel/framework/src/Illuminate/Support/Arr.php:175 { …}
    ./vendor/laravel/framework/src/Illuminate/Support/Collection.php:671 { …}
    ./vendor/laravel/telescope/src/Watchers/QueryWatcher.php:81 { …}
    ./vendor/laravel/telescope/src/Watchers/QueryWatcher.php:33 { …}
    ./vendor/laravel/framework/src/Illuminate/Events/Dispatcher.php:360 { …}
    ./vendor/laravel/framework/src/Illuminate/Events/Dispatcher.php:209 { …}
    ./vendor/laravel/framework/src/Illuminate/Database/Connection.php:826 { …}
    ./vendor/laravel/framework/src/Illuminate/Database/Connection.php:682 { …}
    ./vendor/laravel/framework/src/Illuminate/Database/Connection.php:635 { …}
    ./vendor/laravel/framework/src/Illuminate/Database/Connection.php:459 { …}
    ./vendor/laravel/framework/src/Illuminate/Database/Connection.php:411 { …}
    ./vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php:2541 { …}
    ./vendor/laravel/telescope/src/Storage/DatabaseEntriesRepository.php:117 { …}
    ./vendor/laravel/telescope/src/Telescope.php:453 { …}
    ./vendor/laravel/telescope/src/ListensForStorageOpportunities.php:44 { …}
    Laravel\Telescope\Telescope::Laravel\Telescope\{closure}() {}
    ./vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php:29 { …}
    ./vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php:75 { …}
    ./vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php:31 { …}
    ./vendor/laravel/framework/src/Illuminate/Container/Container.php:572 { …}
    ./vendor/laravel/framework/src/Illuminate/Foundation/Application.php:965 { …}
    ./vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php:149 { …}
    ./artisan:51 {
      ›
      › $kernel->terminate($input, $status);
      ›
      arguments: {
        $input: Symfony\Component\Console\Input\ArgvInput {#25 …}
        $status: 0
      }
    }
  }
}
Script @php artisan package:discover --ansi handling the post-autoload-dump event returned with error code 1
```

**Comment by 907544030** [comment]
Waiting for release
https://github.com/laravel/telescope/pull/239

**Comment by raynox** [comment]
Same here!

**Comment by kylescousin** [comment]
Any news on this?

**Comment by mmccaff** [comment]
@kylescousin It looks like the fix was merged into the "1.0" branch but not yet master. I installed "dev-master" with composer and after that the issue was resolved.

**Comment by AndreFigueredo-Multilaser** [comment]
Same problem here!

**Comment by jlcanizales** [comment]
Same here! :(

**Comment by themazim** [comment]
same behaviour here.

**Comment by Braunson** [comment]
I reported this issue this morning, still waiting on a new release..

**Comment by DaanAmsterdam** [comment]
looks like it's fixed!

**Comment by themsaid** [comment]
releasing in a bit
